dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md

```diff
@@ -108,15 +108,69 @@ kind 通过 Docker 容器模拟出一个本地的 Kubernetes 集群。kind 的
 
 ## 第 2 步：在 Kubernetes 集群上部署 TiDB Operator
 
-参考[部署 TiDB Operator](/dev/tidb-in-kubernetes/deploy/tidb-operator.md#安装-tidb-operator)中的操作。
+1. 安装 Helm 并配置 PingCAP 官方 chart 仓库，参考 [使用 Helm](/dev/tidb-in-kubernetes/reference/tools/in-kubernetes.md#使用-helm) 小节中的操作。
+2. 部署 TiDB Operator，参考 [安装 TiDB Operator](/dev/tidb-in-kubernetes/deploy/tidb-operator.md#安装-tidb-operator) 小节中的操作。
 
 ## 第 3 步：在 Kubernetes 集群中部署 TiDB 集群
 
-参考[标准 Kubernetes 上的 TiDB 集群](/dev/tidb-in-kubernetes/deploy/general-kubernetes.md#部署-tidb-集群)中的操作。
+参考[在标准 Kubernetes 上部署 TiDB 集群](/dev/tidb-in-kubernetes/deploy/general-kubernetes.md#部署-tidb-集群)中的操作。
 
 ## 访问数据库和监控面板
 
-参考[查看监控面板](/dev/tidb-in-kubernetes/monitor/tidb-in-kubernetes.md#查看监控面板)中的操作。
+通过 `kubectl port-forward` 暴露服务到主机，可以访问 TiDB 集群。命令中的端口格式为：`<主机端口>:<k8s 服务端口>`。
+
+> **注意：**
+>
+> 如果你不是在本地 PC 而是在远程主机上部署的 kind 环境，可能无法通过 localhost 访问远程主机的服务。如果使用 kubectl 1.13 或者更高版本，可以在执行 `kubectl port-forward` 命令时添加 `--address 0.0.0.0` 选项，在 `0.0.0.0` 暴露端口而不是默认的 `127.0.0.1`：
+>
+> {{< copyable "shell-regular" >}}
+>
+> ```shell
+> kubectl port-forward --address 0.0.0.0 -n tidb svc/<release-name>-grafana 3000:3000
+> ```
+
+- 通过 MySQL 客户端访问 TiDB
+
+    在访问 TiDB 集群之前，请确保已安装 MySQL client。
+
+    1. 使用 kubectl 暴露 TiDB 服务端口：
+
+        {{< copyable "shell-regular" >}}
+
+        ``` shell
+        kubectl port-forward svc/<release-name>-tidb 4000:4000 --namespace=<namespace>
+        ```
+
+        > **注意：**
+        >
+        > 如果代理建立成功，会打印类似输出：`Forwarding from 0.0.0.0:4000 -> 4000`。测试完成后按 `Ctrl + C` 停止代理并退出。
+
+    2. 然后，通过 MySQL 客户端访问 TiDB，打开一个**新**终端标签或者一个**新**终端窗口，执行下面的命令：
+
+        {{< copyable "shell-regular" >}}
+
+        ``` shell
+        mysql -h 127.0.0.1 -P 4000 -u root
```

Comment:
If the kind is deployed on remote VM or host, we have to access the service with the VM or host IP, right?

Reply:
PTAL Thanks!